src/scene/shader-lib/chunks/common/frag/msdf.js

```diff
@@ -1,13 +1,7 @@
 export default /* glsl */`
 uniform sampler2D texture_msdfMap;
 
-#ifdef GL_OES_standard_derivatives
 #define USE_FWIDTH
```

Comment:
Can you not just remove `USE_FWIDTH` then, since it's always defined?

Reply:
yep, did that, thanks